docs/csharp/tutorials/microservices.md

```diff
@@ -375,44 +323,60 @@ the machine parameters when you start, you only need to
 supply any changes. The changes here will be to include
 our application.
 
-In this first sample, we'll use the `latest` version of
+In this first sample, we'll use the `1.1-sdk-msbuild` version of
 the dotnet image. This is the easiest way to create a working Docker
 environment. This image include the dotnet core runtime, and the dotnet SDK. 
```

Comment:
This image include the dotnet core runtime, and the dotnet SDK. ->

This image includes the .NET Core runtime and the .NET Core SDK.